### PR TITLE
Added - Niran N - Compiled Successfully when run

### PR DESCRIPTION
**Pull request by felirox**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="GDSC Lead Map"
```
Comment (aravindvnair99) [inline public/index.html:9]:
```suggestion
      name="description"
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@felirox Please submit a clean PR with only the map data. Follow the steps mentioned in the [contributing guidelines](https://github.com/abhinavsri360/GDSC-Lead-Map/blob/main/CONTRIBUTING.md).

File: public/index.html (hunk 1 above)
Comment (abhinavsri360) [reply public/index.html:9]: Shouldn't the name be GDSC Lead Map?

**Review by abhinavsri360 — COMMENTED** [review]
Moreover, the PR looks good to me tho, what do you say @aravindvnair99

File: public/index.html (hunk 1 above)
Comment (aravindvnair99) [reply public/index.html:9]: @abhinavsri360 He changed the `meta` property name. `GDSC Lead Map` is not a valid `meta` property.

Comment (abhinavsri360) [reply public/index.html:9]: Ohh, I didn't know that before, sorry 👀

**Comment by aravindvnair99** [comment]
> Moreover, the PR looks good to me tho, what do you say @aravindvnair99

@abhinavsri360 He changed the `meta` property name. `GDSC Lead Map` is not a valid `meta` property.

Refer: https://html.spec.whatwg.org/multipage/semantics.html#the-meta-element

Also, he added a 257 character description. Ideally, it should be 120 max.

File: public/index.html (hunk 1 above)
Comment (felirox) [reply public/index.html:9]: Oops, my bad :P
I'll submit a clean PR

**Pull request closed without merging**